src/sfctl/helps/main.py

```diff
@@ -93,3 +93,28 @@
     type: group
     short-summary: Store and query properties under Service Fabric names
 """
+
+helps['mesh'] = """
+    type: group
+    short-summary: Delete and manage Service Fabric Mesh applications
+"""
+
+helps['mesh app'] = """
+    type: group
+    short-summary: Get and Delete application resources
+"""
+
+helps['mesh service'] = """
+    type: group
+    short-summary: Get details or List services of an application resource
+"""
+
+helps['mesh service-replica'] = """
+    type: group
+    short-summary: Get details or List replicas of a given service in an application resource
```

Comment:
Get replica details and list replicas of a given service in an application resource